### ISSUE TITLE
Files deleted in a send-only folder after "Override Changes" from another device

### ISSUE THREAD

**Issue by vnks** [syncthing/syncthing#7721]
I have a setup with a receive-only folder, and multiple send-only devices syncing to it. The idea is to have a central backup location for files produced on these devices. This is exactly what send-only folders are supposed to be, per [documentation](https://docs.syncthing.net/users/foldertypes.html#send-only-folder):

> The intention is for this to be used on devices where a “reference copy” of files are kept - where the files are not expected to be changed on other devices or where such changes would be undesirable.

However, after clicking "Override changes" on the other device with this send-only folder, both "send-only" folder now have the same content, and the files on the other device were deleted - that data is lost and I have no way to recover it.  The UI says that files in send-only folders are "protected from changes made on other devices", yet in this case the only copies of the files were deleted. It happened on an Android Syncthing client 1.16.0.

To summarize again: 3 nodes, one folder, receive-only on one node, send-only on the other two. Different state on send-only node, after clicking "Override Changes" both send-only devices have the same files, with files unique to the non-overriden send-only device deleted.

**Comment by AudriusButkevicius** [comment]
I think this should be first discussed on the forum to get to the bottom of the situation, as based on what you are describing it does not look like the folder was of the type you are describing.

I suggest you try to reproduce this in isolation on a fake setup and if you can still repro, open a thread on the forum.

**Comment by vnks** [comment]
Took a look through the logs and you're right, looks like operator error. Looks like I had the folder set to "send and receive" and it synced before I changed it to "send only".
